frontend/src/components/store/PluginCard.tsx

```diff
@@ -137,77 +128,82 @@ const PluginCard: FC<PluginCardProps> = ({ storePlugin, installedPlugin }) => {
+                    t('PluginLoader.plugin_uninstall.title', { name: installedPlugin.name }),
+                    t('PluginLoader.plugin_uninstall.button'),
+                    t('PluginLoader.plugin_uninstall.desc', { name: installedPlugin.name }),
+                  );
+                }}
               >
-                <ButtonItem
-                  bottomSeparator="none"
-                  layout="below"
-                  onClick={() =>
-                    requestPluginInstall(storePlugin.name, storePlugin.versions[selectedOption], installType)
-                  }
-                >
-                  <span
-                    className="deckyStoreCardInstallText"
-                    style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', gap: '5px' }}
-                  >
-                    {installType === InstallType.UPDATE ? (
-                      <>
-                        <FaArrowUp /> {t('PluginCard.plugin_update')}
-                      </>
-                    ) : installType === InstallType.REINSTALL ? (
-                      <>
-                        <FaRecycle /> {t('PluginCard.plugin_reinstall')}
-                      </>
-                    ) : installType === InstallType.DOWNGRADE ? (
-                      <>
-                        <FaArrowDown /> {t('PluginCard.plugin_downgrade')}
-                      </>
-                    ) : installType === InstallType.OVERWRITE ? (
-                      <>
-                        <FaDownload /> {t('PluginCard.plugin_overwrite')}
-                      </>
-                    ) : (
-                      // installType === InstallType.INSTALL (also fallback)
-                      <>
-                        <FaDownload /> {t('PluginCard.plugin_install')}
-                      </>
-                    )}
-                  </span>
-                </ButtonItem>
-              </div>
-              <div className="deckyStoreCardVersionContainer" style={{ minWidth: '130px' }}>
-                <Dropdown
-                  rgOptions={
-                    storePlugin.versions.map((version, index) => ({
-                      data: index,
-                      label: (
-                        <div style={{ display: 'flex', alignItems: 'center', gap: '5px' }}>
-                          {version.name}
-                          {installedPlugin && installedVersionIndex === index ? <FaCheck /> : null}
-                        </div>
-                      ),
-                    })) as SingleDropdownOption[]
-                  }
-                  menuLabel={t('PluginCard.plugin_version_label') as string}
-                  selectedOption={selectedOption}
-                  onChange={({ data }) => setSelectedOption(data)}
-                />
-              </div>
-            </Focusable>
-          </PanelSectionRow>
-        </div>
+                <FaTrash />
+              </DialogButton>
+            ) : null}
```

Comment:
Changed these `ButtonItem` into raw `DialogButton`. Is it okay?

As far as I can tell `ButtonItem` is geared towards standalone fields/actions on panels, but it has a lot of styling to support that use case which does not apply here. It was also not very style-able (was being a PITA to actually style properly here into a row of buttons with different sizes) and there was a lot of code around just trying to undo the field-like styling.

This seems to match the code style on the settings panel, but I'm not sure if I'm missing some reason to use `ButtonItem` in the store case.

Reply:
this is probably fine it just used button item because store was implemented really early on